lib/MySQL_Monitor.cpp

```diff
@@ -1605,16 +1612,14 @@ void * monitor_galera_thread(void *arg) {
 				}
 				goto __end_process_galera_result2;
 			}
-/*
 			if (num_fields!=7) {
 				proxy_error("Incorrect number of fields, please report a bug\n");
-				goto __end_process_galera_result;
+				return nullptr;
```

Comment:
Leaving `goto _end_process_galera_result;` , we are actually able to log the event in the monitor table.
I would revert this.